tests/NimblePros.MediatR.Contrib.Test/Behaviors/LoggingBehaviorTests.cs

```diff
@@ -0,0 +1,58 @@
+﻿using Microsoft.Extensions.Logging;
+
+namespace NimblePros.MediatR.Contrib.Test.Behaviors;
+
+public class LoggingBehaviorTests
+{
+  [Fact]
+  public async Task Should_ReturnSuccess_GivenValidInput()
+  {
+    // Arrange
+    var nextMock = new Mock<INext>();
+    var loggerMock = new Mock<ILogger<TestCommand>>();
+    var command = new TestCommand() { Name = "Test name" };
+    var loggingBehavior = new LoggingBehavior<TestCommand, bool>(loggerMock.Object);
```

Comment:
Pull common mocks up to fields. Since command varies you can keep it local or make it a field and assign it - either way is fine for that one.